WordPress/Classes/Services/PageCoordinator.swift

```diff
@@ -3,15 +3,7 @@ import Foundation
 class PageCoordinator {
     typealias TemplateSelectionCompletion = (_ layout: PageTemplateLayout?) -> Void
 
-    static func showLayoutPickerIfNeeded(from controller: UIViewController, forBlog blog: Blog, completion: @escaping TemplateSelectionCompletion) {
-        if FeatureFlag.gutenbergModalLayoutPicker.enabled && blog.isGutenbergEnabled {
```

Comment:
```
blog.isGutenbergEnabled
```

I think we might want to keep this part of the check in-place to prevent showing the layout picker when using the classic editor.

Reply:
Good catch! I'll update it. Thanks!